packages/desktop/pages/recording.tsx

```diff
@@ -0,0 +1,232 @@
+
+  useEffect(() => {
+    async function checkAudioDevices() {
+      if (window.wowarenalogs.obs.getAudioDevices) {
+        const devices = await window.wowarenalogs.obs.getAudioDevices();
+        setOutputAudioOptions(devices?.output || []);
+      }
+      if (window.wowarenalogs.obs.getConfiguration) {
+        const config = await window.wowarenalogs.obs.getConfiguration();
+        setConfigStore(config);
+      }
+    }
+    checkAudioDevices();
+  }, []);
+
+  useEffect(() => {
+    console.log('config subscribed');
+    if (window.wowarenalogs.obs.configUpdated) {
+      window.wowarenalogs.obs.configUpdated((_e, newConf) => {
+        console.log('new config', newConf);
+        setConfigStore(newConf);
+      });
+    }
+    return () => {
+      console.log('Config unsubscribed');
+      window.wowarenalogs.obs.removeAll_configUpdated_listeners &&
+        window.wowarenalogs.obs.removeAll_configUpdated_listeners();
+    };
+  }, []);
+
+  useEffect(() => {
+    if (window.wowarenalogs.obs.recorderStatusUpdated) {
+      window.wowarenalogs.obs.recorderStatusUpdated((_e, status, err) => {
+        console.log('stat', { status, err });
+        setRecordingStatus(status);
+        setRecordStatusError(err || '');
+      });
+    }
+    return () => {
+      window.wowarenalogs.obs.removeAll_recorderStatusUpdated_listeners &&
+        window.wowarenalogs.obs.removeAll_recorderStatusUpdated_listeners();
+    };
+  }, []);
+
+  const maybeAudioChoice = outputAudioOptions.find((a) => a.id === configStore?.audioOutputDevices);
+  const maybeAudioDevice = maybeAudioChoice?.description || 'No device found!';
+
+  const showDebugInfo = process.env.NODE_ENV === 'development' && clientContext.isDesktop;
+
+  return (
+    <div className="p-2 gap-2 min-h-screen">
+      <div className="text-2xl font-bold mb-2">OBS Recording Settings</div>
+      <div className="flex flex-row justify-between">
+        <div className="flex flex-col gap-2">
+          <div className="flex flex-row gap-2 items-center">
+            {recStates[recordingStatus] && recStates[recordingStatus].icon}
+            {recStates[recordingStatus] && recStates[recordingStatus].message}
+            {recordStatusError}
+          </div>
+          <div>
+            <div className="font-bold">Audio Output Device:</div>
+            <div className="flex flex-row gap-2 items-center">
+              <Dropdown
+                menuItems={outputAudioOptions.map((k) => ({
+                  onClick: () => {
+                    window.wowarenalogs.obs?.setConfig && window.wowarenalogs.obs.setConfig('audioOutputDevices', k.id);
+                  },
+                  key: k.id,
+                  label: k.description,
+                }))}
+              >
+                <TbCaretDown size={20} />
+              </Dropdown>
+              <div>{maybeAudioDevice}</div>
+            </div>
+          </div>
+          <div>
+            <div className="font-bold">Resolution:</div>
```

Comment:
feels like this page should exist in a "settings" tab instead of "recording" as its own top level tab

Reply:
Yeah certainly could be - I don't have a super strong opinion on this

Reply:
since we already have a settings page, feels like it's better to consolidate all settings there which educates people about other settings as well. Having a recording page makes it feel like recording needs to be manually controlled which is not the case - this page is really just about "recording related settings", right?